default/templates/filters/embed-image.tpl

```diff
@@ -1 +1 @@
-<img class="mb-3" src="${ema:url}" alt="${ema:alt}" />
\ No newline at end of file
+<img class="mb-3" src="${ema:url}" alt="${ema:alt}" style="display: inline;" />
```

Comment:
You can replace this with Tailwind's [`inline`](https://tailwindcss.com/docs/display#inline) class. 

More generally we should use Tailwind's classes wherever applicable, and resort to `style` attribute only when absolutely necessary. Tailwind is a feature of the default templates (you can use them index.yaml for instance).